main/ui/src/main/java/org/cryptomator/ui/unlock/UnlockModule.java

```diff
@@ -35,7 +35,10 @@
 
 	private static final Logger LOG = LoggerFactory.getLogger(UnlockModule.class);
 
-	public enum PasswordEntry {PASSWORD_ENTERED, CANCELED}
+	public enum PasswordEntry {
+		PASSWORD_ENTERED,
+		CANCELED
+	}
```

Comment:
Is there a rule to keep enums in a single line that...:
* have a no-args constructor
* no methods
* less than n values
* are defined inside of another class

In other words: That are _simple_?

Reply:
Nah, the current setting is fine then.

Reply:
Then it will stay with "Wrap always".

Marking as resolved.